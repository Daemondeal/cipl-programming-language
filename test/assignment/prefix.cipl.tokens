Token          Lexeme         Line 
LET            let            1    
IDENTIFIER     a              1    
EQUAL          =              1    
STRING         "a"            1     a
LINE_END       \n             2    
NOT            not            2    
IDENTIFIER     a              2    
EQUAL          =              2    
STRING         "value"        2     value
LINE_END       \n             2    
EOF                           2    
