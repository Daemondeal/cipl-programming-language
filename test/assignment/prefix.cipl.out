::stdout::

::stderr::
Error on line 2 at `=`: Invalid assignment target.
